airbyte-integrations/connectors/source-news-api/source_news_api/spec.yaml

```diff
@@ -0,0 +1,181 @@
+documentationUrl: https://docs.airbyte.com/integrations/sources/news-api
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: News Api Spec
+  type: object
+  required:
+    - api_key
+    - country
+    - category
+    - sort_by
+  additionalProperties: true
+  properties:
+    api_key:
+      type: string
+      description: API Key
+      airbyte_secret: true
+      order: 0
+    search_query:
```

Comment:
this shouldn't be called `terms`?

Reply:
I think `query` works better, as it allows for boolean operators within the query. Will add an example in the `sample_config`

Reply:
Can you add the example in the examples?

Reply:
Done!